File: markdowner-core/src/main/java/org/jesperancinha/parser/markdowner/badges/model/BadgeSettingGroup.java
package org.jesperancinha.parser.markdowner.badges.model;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Value;

import java.util.List;

@Value
@Builder
@AllArgsConstructor
public class BadgeSettingGroup {
    BadgeType badgeType;
    List<BadgePattern> badgeSettingList;
}

File: markdowner-core/src/main/kotlin/org/jesperancinha/parser/markdowner/badges/model/BadgeSettingGroup.kt
package org.jesperancinha.parser.markdowner.badges.model

import lombok.AllArgsConstructor
import lombok.Builder
import lombok.Value

@Value
@Builder
@AllArgsConstructor
class BadgeSettingGroup {
    var badgeType: BadgeType? = null
    var badgeSettingList: List<BadgePattern>? = null
}